pkg/metrics/resource.go

```diff
@@ -9,12 +9,14 @@ import (
 )
 
 const (
-	resourceMatchPolicyDurationMetricsName = "resource_match_policy_duration_seconds"
-	resourceApplyPolicyDurationMetricsName = "resource_apply_policy_duration_seconds"
-	policyApplyAttemptsMetricsName         = "policy_apply_attempts_total"
-	syncWorkDurationMetricsName            = "binding_sync_work_duration_seconds"
-	syncWorkloadDurationMetricsName        = "work_sync_workload_duration_seconds"
-	policyPreemptionMetricsName            = "policy_preemption_total"
+	resourceMatchPolicyDurationMetricsName  = "resource_match_policy_duration_seconds"
+	resourceApplyPolicyDurationMetricsName  = "resource_apply_policy_duration_seconds"
+	policyApplyAttemptsMetricsName          = "policy_apply_attempts_total"
+	syncWorkDurationMetricsName             = "binding_sync_work_duration_seconds"
+	syncWorkloadDurationMetricsName         = "work_sync_workload_duration_seconds"
+	policyPreemptionMetricsName             = "policy_preemption_total"
+	cronFederatedHPADurationMetricsName     = "cron_federated_hpa_process_duration_seconds"
+	cronFederatedHPARuleDurationMetricsName = "cron_federated_hpa_rule_process_duration_seconds"
```

Comment:
```suggestion
	cronFederatedHPADurationMetricsName     = "cronfederatedhpa_process_duration_seconds"
	cronFederatedHPARuleDurationMetricsName = "cronfederatedhpa_rule_process_duration_seconds"
```
Do you think this makes it better?